packages/calling-stateful-client/src/CallAgentDeclarative.test.ts

```diff
@@ -77,7 +79,13 @@ class MockCallAgent implements CallAgent {
     return call;
   }
   join(groupLocator: GroupLocator, options?: JoinCallOptions): Call;
+  /* @conditional-compile-remove-from(stable) @azure/communication-calling1.4.2-beta.1 sdk updates */
+  join(groupChatCallLoctor: GroupChatCallLocator, options?: JoinCallOptions): Call;
```

Comment:
Wait - a `GroupChatCallLocator`! -- perhaps we should use this for our callwithchat composite

Reply:
(filed a work item for me to check this)

Reply:
(filed a work item on me to double check this)